lib/color.zsh

```diff
@@ -4,10 +4,10 @@ prompt_geometry_colorize() {
 }
 
 prompt_geometry_hash_color() {
-  colors=(2 3 4 6 9 12 14)
+  colors=(`seq 1 9`)
```

Comment:
It's defined inside (). Doing `colors=($(seq 1 9))` can seem a bit confusing. I'm fine with both.